contrib/debian/rules

```diff
@@ -214,7 +219,7 @@ override_dh_fixperms:
 	chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/local-listeners
 
 	# systemd-journal
-	# chmod 4750 $(TOP)/usr/libexec/netdata/plugins.d/systemd-journal.plugin
+	chmod 4750 $(TOP)-plugin-systemd-journal/usr/libexec/netdata/plugins.d/systemd-journal.plugin
```

Comment:
Do we need setuid here?

Reply:
Standard practice in DEB packages is to mark files that can use either SUID or capabilities SUID in the package itself, and then try to set up capabilities in the post-install script. This way if setting the capabilities fails, things will fall back gracefully.